src/commands/context/selectDockerContext.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I feel like this file should be in `src/utils`, or maybe in `dockerContextManager`, since it isn't itself a command

Reply:
I was struggling with this too. It is not a general-purpose utility, so `src/utils` is not the best place IMO. It involves UI, so I did not want to put it in `dockerContextManager`. That is why the file ended up together with the rest of context command code.